docs/using.md

```diff
@@ -266,6 +265,40 @@ These are not configurable at the moment and are different for each driver and t
 | VMWare Fusion | OSX | /Users | /Users |
 | Xhyve | OSX | /Users | /Users |
 
+**Note:** Host folder sharing is not implemented in the KVM and Hyper-V driver yet. You can however
+[mount a CIFS-based shared folder](mounting-custom-shared-folders) inside the virtual machine. 
+
+### Mounting custom shared folders
```

Comment:
:+1: The change itself looks good, but I guess it depends on a release of the b2d image. We merged the ifs patch, but did not do a release nor did we update the default image in Minishift.